pages/support.html

```diff
@@ -36,22 +36,15 @@ <h2>{% trans "Community Forums" %}</h2>
   </div>
 
   <div class="halfbox_right">
-    <h2>{% trans "Developer Mailing List" %}</h2>
+    <h2>{% trans "Zulip Chat" %}</h2>
     <p>
-      {% blocktrans with mailing_list_link="https://lists.sourceforge.net/lists/listinfo/mixxx-devel" sourceforge_archive_link="http://sourceforge.net/mailarchive/forum.php?forum=mixxx-devel" gmane_archive_link="http://news.gmane.org/gmane.comp.multimedia.mixxx.devel" %}The <a href="{{ mailing_list_link}} ">mixxx-devel</a> developer mailing list (<a href="{{ sourceforge_archive_link }}">SourceForge</a> and <a href="{{ gmane_archive_link }}">Gmane</a> archives) is the primary means of developer communication.  The mailing list is where the Mixxx team discusses upcoming changes and programming related stuff.{% endblocktrans %}
+      {% blocktrans with zulip_web_link="https://mixxx.zulipchat.com/" zulip_app_link="https://zulipchat.com/apps/" %}If you don't need support, Zulip chat is still a fun place to hang out with the developers and meet some fellow DJs. Join the discussion through <a href="{{ zulip_web_link }} ">Zulip web chat</a> or download the <a href="{{ zulip_app_link }}">Zulip app</a> and configure it to use the server mixxx.zulipchat.com{% endblocktrans %}
```

Comment:
This reads oddly now because it starts with "If you don't need support", but there is no mention of support as there was in the first sentence of the IRC section.